### ISSUE TITLE
[Caffe2] How can I use detectron with pytorch?

### ISSUE THREAD

**Issue by curry111** [pytorch/pytorch#14262]
Hi guys :sweat_smile:  :sweat_smile:  I have a question but I don't know if this the right place to ask or not.
I am using pytorch and I wanted to use detectron and they are saying that I have to install caffe2 first and I did so but when type `from caffe2.python import core`, I see this 
`WARNING:root:This caffe2 python run does not have GPU support. Will run in CPU only mode.
CRITICAL:root:Cannot load caffe2.python. Error: /lib64/libc.so.6: version 'GLIBC_2.14' not found (required by /home/qwe123/data/conda_packages/lib/python3.6/site-packages/caffe2/python/caffe2_pybind11_state.cpython-36m-x86_64-linux-gnu.so)` 
I tried other solutions I found on here but I could not get this to work. I only installed caffe2 because I wanted to use detectron, Can I use it with pytorch without the need of caffe2? My version of pytorch of 4.0, Should I download pytorch version 1.0 to use detectron with it?

**Comment by t-vi** [comment]
I don't know much about the detectron, but I think https://github.com/facebookresearch/maskrcnn-benchmark/ would implement a very similar thing on PyTorch. It does require recent PyTorch (at least 1.0 preview, probably nightly/master).

**Comment by curry111** [comment]
@t-vi thank you so much for your response :smile: I installed pytorch 1.0 but when I imported it in python, I saw this error
`File "<stdin>", line 1, in <module> `                                                                 
`File "/home/qwe123/data/conda_packages/lib/python3.6/site-packages/torch/__init__.py",` `line 84, in 
 <module>`
    `from torch._C import *`
`ImportError: /lib64/libc.so.6: version 'GLIBC_2.14' not found (required by /home/qwe123/data/conda_packages/lib/python3.6/site-packages/torch/lib/libshm.so)`

**Comment by t-vi** [comment]
Yeah, your Linux system is basically from 2010 (as RHEL6), that won't cut the mustard, unfortunately.
